etc/env/example.env

```diff
@@ -197,3 +199,47 @@ DATASETS_SERVICE_PORT=8880
 AUTHORIZATION_SERVICE_VERSION=v0.0.1-alpha
 AUTHORIZATION_SERVICE_HOST=0.0.0.0
 AUTHORIZATION_SERVICE_PORT=7000
+
+
+# keycloak service
+CONTAINER_NAME_CANDIG_AUTH=keycloak
+KC_ADMIN_USER=kcadmin
+KC_ADMIN_PW=admin
+KC_TEST_USER=bob
+KC_TEST_PW=bobspass123
```

Comment:
embedding secrets in global variables is not a recommended practice

Reply:
this is the example env, though...